src/Pages/Index.cshtml

```diff
@@ -46,8 +46,8 @@
                 </div>
                 <div class="col-lg-4 feature-box">
                     <p><i class="fas fa-bullseye fa-4x icon-features"></i></p>
-                    <h3 class="header-features">Goals</h3>
-                    <p>Enter goal by yourself or just provide name and we will find price for you.</p>
+                    <h3 class="header-features">GoalModel</h3>
+                    <p>Enter goal by yourself or just provide name and we will find amount for you.</p>
```

Comment:
Yup, you're correct, when I was renaming database models I accidentally left the box for string renaming checked as well, so that's how all of these errors are here. I tried to catch them, but I as you can see I missed a few. Thanks a lot for pointing them out 👍